src/components/signup/SignupState.js

```diff
@@ -327,11 +304,18 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
           // re-throw exception
           throw exception
         })
+        identifyWith(null, goodWallet.getAccountForType('login'))
+        fireSignupEvent('STARTED', { source })
 
         // for QA
         global.wallet = goodWallet
+        if (torusUserFromProps.privateKey) {
+          log.debug('skipping ready initialization (already done in AuthTorus)')
+          await API.ready
+          return { goodWallet, userStorage }
+        }
 
-        fireSignupEvent('STARTED', { source })
+        const login = retryImport(() => import('../../lib/login/GoodWalletLogin'))
```

Comment:
```suggestion
        await retryImport(() => import('../../lib/login/GoodWalletLogin'))
```

Reply:
its more clear where in the next line i do
login.then() ....
instead of retryimpot(.....................).then()...